src/main/scala/problems/Mux4.scala

```diff
@@ -42,4 +42,8 @@ class Mux4 extends Module {
 
 
   //Implement above ----------
+
+
+  // make the compile process happy, needs to be substituted by the solution
+  io.out := 1.U
```

Comment:
Since the comments say "Implement above" just a few lines above, this could be confusing if accidentally left in place. One possible idea is `io.out := ???` between "Implement below" and "Implement above".
Maybe leaving it as `1.U` is fine, but perhaps put this just above?